airbyte-webapp/src/pages/SourcesPage/pages/SourceItemPage/components/DeleteSource.tsx

```diff
@@ -43,6 +45,7 @@ const DeleteSource: React.FC<IProps> = ({
   const connectionDelete = useFetcher(ConnectionResource.deleteShape());
 
   const onDelete = async () => {
+    afterDelete();
```

Comment:
SHouldn't that be called at the end of onDelete?

Reply:
Good point!